src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1200,16 +1200,20 @@ public function resolveServices($value)
      *     }
      * }
      *
-     * @param string $name The tag name
+     * @param string $name
+     * @param bool   $throwOnAbstract
      *
      * @return array An array of tags with the tagged service as key, holding a list of attribute arrays
      */
-    public function findTaggedServiceIds($name)
+    public function findTaggedServiceIds($name, $throwOnAbstract = false)
```

Comment:
this is mostly internal, so not sure it's worth it
but please open a PR if you want to work on it